Makefile

```diff
@@ -29,8 +29,8 @@ v1alpha1/proto/grafeas.pb.go: .install.protoc-gen-go v1alpha1/proto/grafeas.prot
 		-I ./ \
 		-I vendor/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis \
 		-I vendor/github.com/googleapis/googleapis \
-		--go_out=./ \
-		v1alpha1/proto/grafeas.proto
+	    v1alpha1/proto/grafeas.proto \
```

Comment:
Are these lines meaning to be switched around?

Reply:
Yes - following the grpc instructions from https://grpc.io/docs/quickstart/go.html